cloud-common/src/main/java/com/sequenceiq/cloudbreak/poller/StatusCheckerTask.java

```diff
@@ -1,4 +1,4 @@
-package com.sequenceiq.cloudbreak.service;
+package com.sequenceiq.cloudbreak.poller;
 
 public interface StatusCheckerTask<T> {
```

Comment:
There is a PollTask.java in the cloud-api, please use that here, instead of introducing StatusCheckerTask